### DISCUSSION TITLE
best way to enable/disable HyeperHDR between TV watching?

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperHDR#357] [Q&A]
I'm using HyperHDR, Home Assistant and the HyperHDR integration from mjoshd.

I have tried to use an automation, when I turn on the TV that it should enable the HyperHDR as well, or turn on/off the LED.
It works sometimes and sometimes the LED is just white light and I need to reboot the RPi.

I wonder what should I enable/disable, is it OK to just enable disable the the LED Device in "remote control" or should I enable/disable the "LED Hardware Instance" ?

**Comment by awawa-dev** [comment]
1) what is your LED driver? only one instance?
2) UDP based LED drivers are not guaranteed to be turned off nicely. Additionally Philips Hue devices are not guaranteed to be turned off even if the Hue bridge receives the command: the is no guarantee that particular lamp will receive the command from the bridge either.
3) You should only use commands that are available in the official API (JSON API tab in the advanced menu). There is no such command that can disable particular instance, only all instances and it's behaves differently than disabling single instance in the general tab by a hand (which should be avoided)
4) if you disabled the background effect and if there is no video source, the LED should be disabled too. So you can disable the video grabber in the grabber control settings, and enable/disable it using remote JSON API when the TV is on.

**Reply by DariBer** [reply]
oh, sorry I missed that, I'm using HyperSerial 8266 and my LED is SK6812. Only one instance.

**Comment by DariBer** [comment]
can I use the "Remote Control"?
 when enabling/disbling the USB Grabber, it looks like it also anele/disable the LED Device, that's OK?

**Reply by awawa-dev** [reply]
Yes, that's exactly how it should behave when the background effect is disabled and no video source is available. You can use "Remote control" if you like: it's a fronted to JSON API anyway.

**Reply by awawa-dev** [reply]
BTW you are using v18?

**Reply by DariBer** [reply]
yes, it's v18 I'm using
